devito/interfaces.py

```diff
@@ -24,6 +24,16 @@
 # manipulation with the correct shapes, pointers, etc.
 _SymbolCache = {}
 
+# This global stores whether newly allocated memory should be initialised
+# using Devito's own JIT engine. It will otherwise be initialised using
+# numpy.
+Global_First_Touch = None
+
+def set_global_first_touch(flag):
```

Comment:
In [parameters](https://github.com/opesci/devito/blob/451849e46a50114215e875680a262665fdceddd2/devito/parameters.py#L153), as is the convention for the other configuration parameters.